sdks/python/apache_beam/transforms/sql_test.py

```diff
@@ -19,6 +19,15 @@
 
 # pytype: skip-file
 
+# beam-playground:
+#   name: SQLTest
+#   description: Unit-test for the transforms that use
+#     the SQL Expansion service.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Beam SQL
```

Comment:
have you tested that these examples run well on the playground?

Reply:
Hi Pablo, yes, all these examples have been tested and work well